### PR TITLE
SLES-12-030310, SLES-15-010410 'Ensure real-time clock is set to UTC'

### PR DESCRIPTION
**Pull request by brett060102**
#### Description:

- Add rule, remediation, and test for STIGS SLES-12-030310 and SLES-15-010410
- 'Ensure real-time clock is set to UTC'

#### Rationale:

- If time stamps are not consistently applied and there is no common time reference, it is difficult to perform forensic analysis.

### CODE REVIEW

File: linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+   {{{ oval_metadata("Ensure RTC is using UTC as its time base") }}}
+    <criteria operator="AND">
+      <criterion comment="Check softlink exists for /etc/localtime and look for UTC pattern" test_ref="test_ensure_rtc_utc_configuration" />
+    </criteria>
+  </definition>
+
+  <unix:symlink_test check="all" check_existence="all_exist" id="test_ensure_rtc_utc_configuration" comment="Ensure softlink exist for localtime with UTC pattern" version="1">
+    <unix:object object_ref="object_ensure_rtc_utc_configuration" />
+    <unix:state state_ref="object_ensure_symlink_utc_configuration" />
+  </unix:symlink_test>
```
Comment (yuumasato) [inline linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml:12]: The symlink_test is a new test in OVAL 5.11 test, could you wrap the whole definition in a Jinja conditional?
Similar to https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_sufficiently_large_partition/oval/shared.xml#L1

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml (hunk 1 above)
Comment (brett060102) [reply linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml:12]: Done, hope it looks right.

**Comment by brett060102** [comment]
@yuumasato Is this one OK now?

File: linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml:13]: Apologies for my delay, I believe the version should be just `5.11`.

Comment (brett060102) [reply linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml:13]: Does this look right? And sorry for my confusion.

Comment (yuumasato) [reply linux_os/guide/system/logging/ensure_rtc_utc_configuration/oval/shared.xml:13]: @brett060102 Yes, thanks.

**Comment by brett060102** [comment]
@yuumasato Thank you.

**Pull request merged**